recipes/polyscope/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "polyscope" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "eceecaf3762f1b785726dab2696d23dfd353a43adac032924066a596ebc711b3"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv --install-option=\"--exclude-arch\""
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+  host:
+    - python
+    - numpy
```

Comment:
Does this link against numpy? If it does see: https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

If it doesn't, this can probably be removed from `host`:
```suggestion
```

Reply:
Ah I see, done!